fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -79,7 +79,7 @@ def self.show_details(filter: nil)
         if !filter.nil? && filter.length > 1
           candidates = []
           all_actions(nil) do |a, action_name|
```

Comment:
can we use something different than `a` here, i guess its meant to be the hash key?